test/acceptance/integration/NGSI 10/Delete entity/delete.feature

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+# __author__ = 'Jon Calderin Goñi (jon dot caldering at gmail dot com)'
+@delete_entity
+Feature: Subscribe context tests
+
+  Background:
+    Given the Context Broker started with multitenancy
+
+  Scenario: Delete an entity does not exist with convenience and type extended
+    # Append
+    Given a new NGSI version "10" petition with the service "service" and the subservice "/subservice"
```

Comment:
NTC, it will be checked in https://github.com/telefonicaid/fiware-orion/pull/800